controls/cis_rhel8.yml

```diff
@@ -1277,6 +1277,17 @@ controls:
       - var_nftables_base_chain_priorities=chain_priorities
       - var_nftables_base_chain_policies=chain_policies
 
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5246
+  - id: 3.4.2.2
+    title: Ensure host based firewall loopback traffic is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: pending
```

Comment:
These rules could be used here:
- firewalld_loopback_traffic_trusted
- firewalld_loopback_traffic_restricted

They are also used in RHEL 9.